openedx/features/learner_profile/static/learner_profile/js/learner_profile_factory.js

```diff
@@ -139,7 +145,10 @@
                     valueAttribute: 'bio',
                     helpMessage: '',
                     persistChanges: true,
-                    messagePosition: 'header'
+                    messagePosition: 'header',
+                    maxCharacters: 300,
+                    charCountMessage: gettext('characters remaining'),
```

Comment:
As mentioned before, I'd make this:

    gettext('{character_count} characters remaining')

to allow for languages where the count appears elsewhere in the sentence.